ProcessDoctor/Imaging/Extensions/BitmapExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.Drawing.Imaging;
+using System.IO;
+using Bitmap = Avalonia.Media.Imaging.Bitmap;
+
+namespace ProcessDoctor.Imaging.Extensions;
+
+public static class BitmapExtensions
+{
+    public static Bitmap? ToAvaloniaBitmap(this System.Drawing.Bitmap? nativeBitmap)
+    {
+        if (nativeBitmap is null)
+        {
+            return null;
+        }
+
+        using var stream = new MemoryStream();
+        nativeBitmap.Save(stream, ImageFormat.Bmp);
+        stream.Position = 0;
+
+        return new Bitmap(stream);
```

Comment:
This is not good because it will dispose the stream after creating the bitmap, and it's well known to own the stream and break sometimes if it gets disposed.

Let's remove the `using` here.

Reply:
I don't think that's a problem.
From `Avalonia`'s sources:
[Bitmap.cs](https://github.com/AvaloniaUI/Avalonia/blob/21ad94b98085767674204e7da71dd2b403759c38/src/Avalonia.Base/Media/Imaging/Bitmap.cs#L68)
[ImmutableBitmap.cs](https://github.com/AvaloniaUI/Avalonia/blob/21ad94b98085767674204e7da71dd2b403759c38/src/Skia/Avalonia.Skia/ImmutableBitmap.cs#L24)

The `ImmutableBitmap` is created when `GetFactory().LoadBitmap()` is called. The original `Stream` inside `ImmutableBitmap` is only used during initialization in the constructor (using `SKManagedStream`) and nowhere else.

Are you sure you need to remove `using'?

Reply:
Ah, I missed the fact that this is Avalonia's `Bitmap`, and not the one from `System.Drawing`. I am okay with the current implementation then.